spring-integration-mqtt/src/test/java/org/springframework/integration/mqtt/MqttAdapterTests.java

```diff
@@ -120,21 +110,11 @@ public class MqttAdapterTests {
 		this.alwaysComplete = (IMqttToken) pfb.getObject();
 	}
 
-	@Test
-	public void testCloseOnBadConnectIn() throws Exception {
```

Comment:
Why to remove this test?
Any explanations about other tests removal in this class?

Any chances that you can comment the reason behind changes in the commit message, please?

Reply:
I have mentioned this in the commit message:
```
* Do not close the connection in adapter to be able to use reconnect
```

So these removed test cases just became obsolete. We should not close the client now except in the `destroy` phase.. 
This all has been done to be able to use `connectComplete` callbacks and make the code as similar as possible with minimum additional `if`s.